src/cpp/OpenWsmanClient.cpp

```diff
@@ -195,9 +195,10 @@ string OpenWsmanClient::Get(
 
 string OpenWsmanClient::Get(
 	const string &resourceUri,
-	const NameValuePairs *s) const
+	const NameValuePairs *s,
+	unsigned long flags) const
```

Comment:
Hmm, this changes the API. I'm not sure if this is the right approach or if introducing `OpenWsmanClient::GetWithFlags` would be a better one.

Reply:
Ok, will split to the new functions

Reply:
Done, please look if this fit.

Reply:
Is the new split is ok?